turnip.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_runtime_dependency "rspec", [">=3.0", "<4.0"]
-  s.add_runtime_dependency "cucumber-gherkin", "~> 14.0"
+  s.add_runtime_dependency "cucumber-gherkin", "~> 22.0"
```

Comment:
Currently the latest version of cucumber-gherkin is 24.0.0.
I have confirmed that the test passes with version 24.0.0.
So "~> 24.0" should be used.

Reply:
And in 0-3 months, there will be a `25.0` followed by a `26.0` a few months after that.  #244 will fix the issue for a lot longer. O:)